src/coord/src/coord.rs

```diff
@@ -3840,13 +3851,9 @@ impl Coordinator {
     ) -> Result<ExecuteResponse, CoordError> {
         let catalog = self.catalog.for_session(session);
         let values = sql::plan::plan_copy_from(&session.pcx(), &catalog, id, columns, rows)?;
-
-        let constants = self
-            .prep_relation_expr(values.lower(), ExprPrepStyle::Write)?
-            .into_inner();
-
+        let values = self.view_optimizer.optimize(values.lower())?;
```

Comment:
Does this need the same already optimized fast path as insert?

Reply:
Oh good point. We should just skip optimization here. Thanks, fixed!